tutorials/themes/tutorials-theme/navbar.html

```diff
@@ -0,0 +1,65 @@
+  <form class="input-group col-sm-7 m-auto"  id="search-bar" action="{{ pathto('search') }}" method="get">
+      <div class="input-group-prepend">
+          <button class="input-group-text" type="submit"><i class="fa fa-search fa-fw"></i></button>
+      </div>
+      <input type="text" name="q" id="search-bar-input" class="form-control" placeholder="Search the universe of Astropy" />
+  </form>
+</div>
+
+<!-- Guides, Tutorials, Examples, Documentation Placards -->
+<div class="container-fluid row definition-cards">
+  <div class="card bg-guide col-sm-2 p-0 m-2  ml-auto enable-guides">
+    <div class="card-body">
+      <h2 class="card-title text-center"><a href="{{ pathto('guides') }}" class="text-white">Guides</a></h2>
+      <p class="card-text text-center">Guides are comprehensive, conceptually-focused documents providing stand-alone introductions to core packages in addition to the underlying astronomical concepts.</p>
+    </div>
+  </div>
+  <div class="card bg-tutorial col-sm-2 p-0 m-2">
+    <div class="card-body">
+        <h2 class="card-title text-center"><a href="/tutorials.html?q=filterTutorials" class="text-white">Tutorials</a></h2>
+      <p class="card-text text-center">Tutorials are step-by-step cookbooks for common activities that incorporate several packages. They are more specific and less conceptual than Guides but more extended than Examples.</p>
+    </div>
+  </div>
+  <div class="card bg-documentation col-sm-2 p-0 m-2">
+    <div class="card-body">
+        <h2 class="card-title text-center" style="margin-left: -15px; margin-right: -15px;"><a href="{{ pathto('documentation') }}" class="text-white">Documentation</a></h2>
+      <p class="card-text text-center">Documentation is the complete description of a package with all requisite details, including usage, dependencies, and examples.</p>
+    </div>
+  </div>
+  <div class="card bg-example col-sm-2 p-0 m-2 mr-auto">
+    <div class="card-body">
+        <h2 class="card-title text-center"><a href="{{ pathto('examples') }}" class="text-white">Examples</a></h2>
+      <p class="card-text text-center">Examples are stand-alone code snippets that live in the astropy documentation that demonstrate a specific functionality within a package.</p>
+    </div>
+  </div>      
+</div>
+
+{% else %}
+
+<nav class="navbar navbar-expand-lg bg-navbar">
+  <a class="navbar-brand" href="/"style="padding: 0px 5px;"><img src="../_static/astropy_logo.png" style="height:38px;"></a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor01" aria-controls="navbarColor01" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarColor01">
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item active">
+        <a class="nav-link" href=""http://www.astropy.org/>Astropy<span class="sr-only">(current)</span></a>
```

Comment:
URL is outside of the two quotes

Reply:
Thanks for pointing it out. I have fixed it in the latest commit.